Gemfile

```diff
@@ -1,2 +1,4 @@
 source 'https://rubygems.org'
 gemspec
+
+gem 'rack', '1.6.4'
```

Comment:
Generally we want to use the most optimistic constraint that we can and not pin to a specific version in the Gemfile. Here is how we're handling the rack/ruby version problem in chef:

```
gem "rack", "< 2.0" # 2.0 requires Ruby 2.2+
```

By the way, we are starting soon on getting Chef on Ruby 2.2+